src/sbmlutils/test/report/test_report.py

```diff
@@ -11,10 +11,27 @@
     GLUCOSE_SBML,
     GZ_SBML,
     REPRESSILATOR_SBML,
+    UNCERTAINTY_PATH,
     VDP_SBML,
 )
 
 
+uncertainty_paths = [
```

Comment:
I assume these are the models from the sbml test suite? We should add the complete testsuite models if these are not too big, not only the distrib models.

Reply:
You have to add the models as part of the pull request so the tests run in CI.

Reply:
I took a look at all the models under the ```data``` directory, they aren't too voluminous and I think we can add all of them to the tests. I am thinking of creating a new directory ```testsuite``` under ```test/data``` and copying and renaming each model from the ```data``` directory(further partitioned inside some more directories) to a naming scheme that can be accessed using a for loop. Would this approach be suitable for the task Sir?

Reply:
Please keep the existing models as is. You could add a list for them in the `test/init.py` which you can then use in other tests. Yes add a `test/data/testsuite` for the testsuite.